bin/node/runtime/src/lib.rs

```diff
@@ -1626,4 +1626,14 @@ mod tests {
 
 		is_submit_signed_transaction::<Runtime>();
 	}
+
+	#[test]
+	fn call_size() {
```

Comment:
You should add the same to polkadot/kusama/westend runtimes as well to ensure their size also does not change over time.

Reply:
yes, I'll do in the companion

Reply:
companion should be ready now https://github.com/paritytech/polkadot/pull/3522